stellargraph/datasets/datasets.py

```diff
@@ -850,25 +850,23 @@ def load(self):
         return StellarGraph(nodes=nodes, edges=edges, edge_weight_column="time"), edges
 
 
-@experimental(reason="the data isn't downloaded automatically", issues=[1303])
+@experimental(reason="tests and documentation missing", issues=[1303])
 class METR_LA(
     DatasetLoader,
     name="METR-LA",
     directory_name="METR-LA",
-    url="https://github.com/lehaifeng/T-GCN/tree/master/data",
-    url_archive_format="n/a",
-    expected_files=["los_speed.csv", "los_adj.csv"],
+    url="https://github.com/lehaifeng/T-GCN/archive/master.zip",
```

Comment:
Do you think it's worth pining to a particular commit here, in case there's mutations in future?

```suggestion
    url="https://github.com/lehaifeng/T-GCN/archive/8427128f04157e6fd0b239a8734a468d923cd0c9.zip",
```

(Using the current most recent commit https://github.com/lehaifeng/T-GCN/commit/8427128f04157e6fd0b239a8734a468d923cd0c9)

It's true that we don't have any similar defence for the other datasets, but since it's easy to do here, maybe we should?

(This will likely require updating `expected_files` or `directory_name`.)

Reply:
This does make for a very ugly folder name!: `T-GCN-8427128f04157e6fd0b239a8734a468d923cd0c9`